nldi-liquibase/src/main/resources/liquibase/characteristic_data/tables.sql

```diff
@@ -62,3 +62,30 @@ create table characteristic_data.local_catchment_characteristics
 );
 alter table characteristic_data.local_catchment_characteristics owner to nldi_data;
 --rollback drop table characteristic_data.local_catchment_characteristics
+
+
+--changeset drsteini:create.characteristic_data.plusflowlinevaa_np21
+--preconditions onFail:MARK_RAN onError:HALT
+--precondition-sql-check expectedResult:0 select count(*) from information_schema.tables where table_schema = 'characteristic_data' and table_name = 'plusflowlinevaa_np21'
```

Comment:
Similar question for the tables: why detect table existence in the precondition instead of using a `CREATE TABLE IF NOT EXISTS` statement in the changeset query?